### DISCUSSION TITLE
【求助】9.22升级专业版，支付宝充值，现在还没有升级成功

### DISCUSSION THREAD

**Discussion by zhougit712** [getlantern/lantern-forum-cn#885] [Q&A]
@Mike777Khoo 
9.22 充值，现在还没有升级到专业版，麻烦帮我解决一下，邮箱1018872178@qq.com
![image](https://github.com/user-attachments/assets/46a223cf-d646-4440-b05a-527703214c02)

**Comment by MaxMKMax** [comment]
@zhougit712 好的. 我帮您安排一下.
